src/main/java/core/basesyntax/serviceimpl/ProcessServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ProcessService;
+import core.basesyntax.strategy.Strategy;
+import java.util.List;
+
+public class ProcessServiceImpl implements ProcessService {
+    private final Strategy strategy;
```

Comment:
Ensure that the Strategy dependency is provided using dependency injection principles. If there's a framework like Spring being used in the project, consider annotating this class for dependency injection. If not, ensure that the Strategy instance is passed into this class from the outside (for example, from the Main class) to maintain loose coupling and high cohesion.